### PR TITLE
Non ad group user

### PR DESCRIPTION
**Pull request by amanpdyadav**
## Description

User with ad account but not belonging to any ad groups should show unauthorised message.

[PV-280](https://helsinkisolutionoffice.atlassian.net/browse/PV-280)

## Manual Testing Instructions for Reviewers

Login to admin view with a user that do not have ad groups.

## Screenshots

Screenshot 2022-11-10 at 23 02 59

### CODE REVIEW

**Review by danipran — COMMENTED** [review]
Correct me if I'm wrong, but I recall the initial view having a huge login button... 🧐 
image

File: src/components/MainLayout.tsx [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
+  return (
+    <div className={styles['main-layout']}>
+      <Header />
+      <Divider />
+      <div className={styles['content-container']}>
+        <div className={styles.gutter}>
+          {userRole > UserRole.NON_AD_GROUPS && <Outlet />}
+          {userRole === UserRole.NON_AD_GROUPS && (
+            <div>{t(`message.unauthorized`)}</div>
+          )}
```
Comment (danipran) [inline src/components/MainLayout.tsx:21]: Ah, this is what's causing the login view to not render. If you're not logged in, it renders nothing.

**Review by danipran — CHANGES_REQUESTED** [review]

File: src/components/MainLayout.tsx (hunk 1 above)
Comment (amanpdyadav) [reply src/components/MainLayout.tsx:22]: Fixed it.

**Review by mhieta — APPROVED** [review]
LGTM 👍

**Pull request merged**